SignalServiceKit/src/Messages/Interactions/TSMessage.m

```diff
@@ -236,7 +240,10 @@ - (NSString *)debugDescription
 
 - (NSString *)previewTextWithTransaction:(YapDatabaseReadTransaction *)transaction
 {
-    if ([self hasAttachments]) {
+    if (self.body.length > 0) {
```

Comment:
I'm not sure this is better, since now we exclude the fact that it's an attachment from the notification. 

What about including "\(attachmentDescription): \(caption)" when both are present, something like:

"📽 Attachment: Cha Cha Cha"

Reply:
Reworked this.